src/entries/popup/pages/send/index.tsx

```diff
@@ -111,6 +112,9 @@ export function Send() {
   const { isContact } = useContactsStore();
   const { allWallets } = useWallets();
   const { hiddenAssets } = useHiddenAssetStore();
+  const [urlSearchParams] = useSearchParams();
+
+  const queryToAddress = urlSearchParams.get('to');
```

Comment:
I think that since this can technically be edited by the user we should check if it's a valid address/ens

Reply:
Good catch 🙏